### ISSUE TITLE
support for civicrm 4.5?

### ISSUE THREAD

**Issue by jtbayly** [circleinteractive/uk.co.circleinteractive.offlinerecurring#1]
status?

**Comment by andy-walker** [comment]
I should have something done on this within the next week or two. We have been working on it, but thought it would be better to revise the codebase a bit for 4.5, get it to fit in a bit better within the framework of what Civi 4.5 now provides for recurring payments, which is why it's taking a bit longer than normal to get a version out.

Hope that answers your question.

**Comment by jtbayly** [comment]
Yep. Thanks much. I look forward to it.

**Comment by sleidig** [comment]
Are there any plans to update the extension to CiviCRM 4.5 (or rather to CiviCRM 5.x)?

**Comment by ghost** [comment]
Yes, it's working with 5.67 now. See https://github.com/circleinteractive/uk.co.circleinteractive.offlinerecurring/issues/4 for newer CRM versions. 
Maybe you like to close this issue, @jtbayly ?

**Comment by jtbayly** [comment]
I'm not even sure what this is anymore, so I'll leave it alone and let somebody else test it and close it. :)
